test/modules/Rfq.t.sol

```diff
@@ -66,10 +66,26 @@ contract RfqModuleTest is MatchingBase {
     trades[3] = IRfqModule.TradeData({
       asset: address(option),
       subId: OptionEncoding.toSubId(block.timestamp + 1 weeks, 1700e18, false),
-      price: -400e18,
+      price: 400e18,
       amount: 2e18
     });
+
+    assertEq(subAccounts.getBalance(camAcc, cash, 0), int(cashDeposit));
+
     _submitRfqTrade(trades);
+
+    // option 1: paid 600
+    // option 2: received 600
+    // option 3: received 400
+    // option 4: paid 800
+    assertEq(subAccounts.getBalance(camAcc, cash, 0), int(cashDeposit) - 400e18);
+    assertEq(subAccounts.getBalance(dougAcc, cash, 0), int(cashDeposit) + 400e18);
+
+    assertEq(subAccounts.getBalance(camAcc, option, OptionEncoding.toSubId(block.timestamp + 1 weeks, 1500e18, true)), 2e18);
+    assertEq(subAccounts.getBalance(camAcc, option, OptionEncoding.toSubId(block.timestamp + 1 weeks, 1500e18, false)), -2e18);
+    assertEq(subAccounts.getBalance(camAcc, option, OptionEncoding.toSubId(block.timestamp + 1 weeks, 1700e18, true)), -2e18);
+    assertEq(subAccounts.getBalance(camAcc, option, OptionEncoding.toSubId(block.timestamp + 1 weeks, 1700e18, false)), 2e18);
```

Comment:
One small improvement to just declare the 4 optionSubIds as variables option1 option2 ... instead of encoding it every time.